config/locales/en/errors.yml

```diff
@@ -430,3 +430,4 @@ en:
         non_numerical_minutes: The number of minutes must be a number
         non_integer_hours: The number of hours must be a whole number
         non_integer_minutes: The number of minutes must be a whole number
+        zero_time_period: Enter the number of minutes
```

Comment:
👍 I added this as couldn't see any instances of also having `numericality: { greater_than: 0 }` so eitehr missed or added since... like this better as more self contained so will update messages after confirmation...

Reply:
Sounds good